base/common/ReplxxLineReader.cpp

```diff
@@ -204,7 +206,7 @@ void ReplxxLineReader::openEditor()
         if ((-1 == res || 0 == res) && errno != EINTR)
         {
             rx.print("Cannot write to temporary query file %s: %s\n", filename, errnoToString(errno).c_str());
-            return;
```

Comment:
Bug.

Reply:
Hmm, perhaps we need to `close(fd)` and return here, or else it might execute partial query.

Reply:
That's true.